app/instances.go

```diff
@@ -87,7 +87,7 @@ func (a *App) createTriggers(tConfigs []*trigger.Config, runner action.Runner) (
 			var ok bool
 			ref, ok = support.GetAliasRef("trigger", tConfig.Ref)
 			if !ok {
-				return nil, fmt.Errorf("trigger [%s]'s 'ref' [%s] not imported", tConfig.Id, tConfig.Ref)
+				return nil, fmt.Errorf("incompatible app engine, the app using the trigger [%s] with ref [%s] is not installed", tConfig.Id, tConfig.Ref)
```

Comment:
How about something like:  "unable to start trigger [%s], '%s' not installed", so it would look like: 
```
Failed to create engine: unable to start trigger [receive_http_message], 'github.com/project-flogo/contrib/trigger/rest' not installed.  @mellistibco any thoughts?

```

Reply:
I don't think the error should be 'incompatible app engine'. Perhaps we should also provide a potential solution?

```
Failed to create engine: unable to start trigger [receive_http_message], 'github.com/project-flogo/contrib/trigger/rest' not installed. Use `flogo imports sync` to resolve references.
```

Or something like that?